packages/babel-parser/src/tokenizer/types.ts

```diff
@@ -328,6 +328,9 @@ export const tt = {
   _opaque: createKeywordLike("opaque", { startsExpr }),
   // end: isFlowInterfaceOrTypeOrOpaque
   name: createToken("name", { startsExpr }),
+
+  // placeholder plugin
+  placeholder: createToken("%%", { startsExpr: true }),
```

Comment:
Since this change will also affect `tokenIsLiteralPropertyName`, can you add a parser test for placeholder used as a property name?